### DISCUSSION TITLE
普物实验 TeX & R 急救指北

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#13] [Comments]
# 普物实验 TeX & R 急救指北

这学期选了物理实验B。
以前基本上只在 Markdown 里写 LaTeX，没怎么写过 .tex，R 则完全没有接触过。理论上我应该好好学一学再用，但实际上没时间好好学。总不能用 Word 写实验报告并且手算吧（
这里总结一下我写实验报告用到的 TeX 和 R。

https://ouuan.moe/post/2022/10/basic-tex-and-r-for-physics-lab

**Comment by Chiyuru** [comment]
我倒是觉得处理数据最方便的是一边做一边往 Excel 里敲数据，然后直接用 Excel 的函数处理，再顺便把图画了（

**Reply by ouuan** [reply]
要是原始数据可以是老师在电子表格上签名就好了，手写再敲一遍是真的麻烦（

感觉 R 会更麻烦主要还是因为，没认真学，不太会用，而且也不会统计学，文档都不会查；但如果每次用到的东西差不多的话，以后应该会好一点（

但比起怎么算，我大概更需要一个由计算结果自动生成 TeX 公式。我们还要求写数据代入式，就很麻烦 😢

**Comment by ouuan** [comment]
感觉学物理实验什么物理都没学到，最大的收获是我现在作业和笔记都不用 Markdown 改用 TeX 了（

**Comment by LHRG** [comment]
想问一下 siunitx 宏包的文档在哪里能看到呢？github 上的 README.md 里也没找到[捂脸]

**Reply by LHRG** [reply]
没事了找到了[捂脸]
